internal/core/component.rs

```diff
@@ -82,6 +82,9 @@ pub struct ComponentVTable {
     pub dealloc: unsafe fn(&ComponentVTable, ptr: *mut u8, layout: vtable::Layout),
 }
 
+#[cfg(test)]
+pub(crate) use ComponentVTable_static;
```

Comment:
why is this like this?

Reply:
This is the way to use a macro defined in a different module of the same crate, introduced in rust 1.32 AFAICT.